test/Aggregate/HydratorListenerTest.php

```diff
@@ -80,6 +80,7 @@ public function testOnHydrate(): void
             ->will($this->returnValue($hydrated));
         $event->expects($this->once())->method('setHydratedObject')->with($hydrated);
 
+        /** @psalm-suppress MixedArgumentTypeCoercion */
```

Comment:
Without suppress
```
Error: Argument 1 of Laminas\Hydrator\Aggregate\HydratorListener::onHydrate expects Laminas\Hydrator\Aggregate\HydrateEvent<object>, parent type PHPUnit\Framework\MockObject\MockObject&Laminas\Hydrator\Aggregate\HydrateEvent<object> provided (see https://psalm.dev/194)
```
https://github.com/laminas/laminas-hydrator/runs/3177819016?check_suite_focus=true

This problem is caused from mock. so I just added psalm-suppress.